examples/ab_test.py

```diff
@@ -93,6 +94,70 @@
     "--test_value", type=str, required=True, help="the feature value in test group."
 )
 
+parser.add_argument("--csv", type=str, help="csv output file")
+
+
+def output_results(performance_all, title_list, csv_writer=None):
+    for nproc, performance in performance_all.items():
+        print(
+            " ================ Performance (FPS) NPROC={} ===================================".format(
+                nproc
+            )
+        )
+        title = "Resolution "
+        for t in title_list:
+            title += "\t%-24s" % t
+        print(title)
+        # break down by resolutions
+        for idx in range(len(performance)):
+            row = "%d x %d" % (resolutions[idx], resolutions[idx])
+            # break down by benchmark items
+            for t in title_list:
+                control_fps = performance[idx][t + "_" + str(dr.ABTestGroup.CONTROL)][
+                    "fps"
+                ]
+                test_fps = performance[idx][t + "_" + str(dr.ABTestGroup.TEST)]["fps"]
+                ratio = (
+                    (float(test_fps) - float(control_fps)) / float(control_fps) * 100.0
```

Comment:
I know it is from my original code. But I am now thinking it twice.

To everyone:
Do you think it is better to just compute the speedup instead of the increase rate?
namely:
`speedup = (float(test_fps) / float(control_fps) * 100.0`
According to my current experience, `speedup` is more intuitive (since we always report it as e.g., the speedup is 3.6x)

Reply:
See my below comment. The important thing is we output as an intermediate format. Here's how to compute all metrics everyone wants very easily once this is merged:

```
>>> import pandas as pd
>>> results = pd.read_csv("~/habitat-sim/semantic_bench.csv")
>>> print(results)
   num_procs  resolution   sensor_types    fps_test  fps_control
0          1         128  semantic_only  703.088740   108.107696
1          1         128  rgbd_semantic  173.315447    89.380839
2          1         256  semantic_only  546.703172   104.385620
3          1         256  rgbd_semantic  358.973824    93.043743
>>> results["frame_time_test_ms"] = 1000 / results["fps_test"]
>>> results["frame_time_control"] = 1000 / results["fps_control"]
>>> results["speedup"] = results["fps_test"] / results["fps_control"]
>>> print(results)
   num_procs  resolution   sensor_types    fps_test  fps_control  frame_time_test_ms  frame_time_control   speedup
0          1         128  semantic_only  703.088740   108.107696            1.422296            9.250035  6.503596
1          1         128  rgbd_semantic  173.315447    89.380839            5.769826           11.188080  1.939067
2          1         256  semantic_only  546.703172   104.385620            1.829146            9.579864  5.237342
3          1         256  rgbd_semantic  358.973824    93.043743            2.785718           10.747633  3.858119
>>> filtered_256_px = results[results["resolution"]==256]
>>> print(filtered_256_px)
   num_procs  resolution   sensor_types    fps_test  fps_control  frame_time_test_ms  frame_time_control   speedup
2          1         256  semantic_only  546.703172   104.385620            1.829146            9.579864  5.237342
3          1         256  rgbd_semantic  358.973824    93.043743            2.785718           10.747633  3.858119
>>> avg_256_speedup = filtered_256_px["speedup"].mean()
>>> print(f"avg_256_speedup={avg_256_speedup}")
avg_256_speedup=4.54773034562938
```